utils/docker-ce/files/dockerd.init

```diff
@@ -3,12 +3,92 @@
 USE_PROCD=1
 START=25
 
+DOCKERD_CONF="/tmp/dockerd/daemon.json"
+
+json_add_array_string() {
+	json_add_string "" "$1"
+}
+
+process_config() {
+	local alt_config_file data_root log_level
+
+	rm -f "$DOCKERD_CONF"
+
+	[ -f /etc/config/dockerd ] || {
+		# Use the daemon default configuration
+		DOCKERD_CONF=""
+		return 0
+	}
+
+	config_load 'dockerd'
+
+	config_get alt_config_file globals alt_config_file
+	[ -n "$alt_config_file" ] && [ -f "$alt_config_file" ] && {
+		rm -f "$DOCKERD_CONF"
```

Comment:
This could be removed if you wanted to as it's redundant, it won't exist at this point because of line 15